spipe/pushbits.h

```diff
@@ -1,10 +1,12 @@
 #ifndef _PUSHBITS_H_
 #define _PUSHBITS_H_
 
+#include <signal.h>
```

Comment:
I guess this works, but I'd say that `<sys/types.h>` is a more normal place to find `pthread_t`.